handlers.go

```diff
@@ -111,12 +131,6 @@ func (s *Server) ServeAPI(srv string, logging bool, lsteptok string, cachedir st
 		return nil
 	})
 
-	exporter := esmetrics.Exporter()
```

Comment:
why take this out?

Reply:
Alvin's rebase takes changes from master that were already there. This is unrelated to this PR specifically.